etc/avocado/avocado.conf

```diff
@@ -19,6 +19,8 @@ enabled = True
 installed_packages = False
 # Whether to run certain commands in bg to give extra job debug information
 profiler = False
+# Force LANG for sysinfo collection
+lang = C
```

Comment:
LC_ALL will change everything indeed, by overriding all LC_* variables. In your personal case, if you want only the text messages to be translated, you should be using LC_MESSAGES, not LANG. See locale(1).

My personal example:

```
$ locale
LANG=
LC_CTYPE=pt_BR.UTF-8
LC_NUMERIC=en_US.UTF-8
LC_TIME=pt_BR.UTF-8
LC_COLLATE=pt_BR.UTF-8
LC_MONETARY=pt_BR.UTF-8
LC_MESSAGES=en_US.UTF-8
LC_PAPER=pt_BR.UTF-8
LC_NAME=pt_BR.UTF-8
LC_ADDRESS=pt_BR.UTF-8
LC_TELEPHONE=pt_BR.UTF-8
LC_MEASUREMENT=pt_BR.UTF-8
LC_IDENTIFICATION=pt_BR.UTF-8
LC_ALL=
```